MAUI/DataForm/layout.md

```diff
@@ -0,0 +1,372 @@
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+
+    <dataForm:SfDataForm
+        x:Name="dataForm"
+        ColumnCount="2"
+        >
+    </dataForm:SfDataForm>
+
+</ContentPage>
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.ColumnCount = 2;
+
+{% endhighlight %}
+{% endtabs %}
+
+N> Setting the `ColumnCount` property to SfDataForm does not arrange the data field in a group according to the column count. To set the column count for data fields in the data form group, refer to [loading different layout for data form group]
+
+When the label position is Left, the grid layout is shown as follows:
+
+When the label position is Top, the grid layout is shown as follows:
+
+## Label visibility
+
+The label can be hided by defining the [DataFormDisplayOptions] attribute or by handling `GenerateDataFormItem` event. In this case, only the editor will be loaded.
+
+#### Using attributes
+
+{% tabs %}
+{% highlight C# %}
+private double? percentage;
+
+    [DataFormDisplayOptions(ShowLabel = false)]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Using event
+
+{% tabs %}
+{% highlight C# %}
+
+ this.dataForm.GenerateDataFormItem += OnGenerateDataFormItem;
+
+    private void OnGenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Name")
+        {
+            e.DataFormItem.PlaceholderText = "Enter your name";
+            e.DataFormItem.ShowLabel = false;
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Label position
+
+Label can be positioned either top or left side of the editor. By using the [DataFormDefaultLayoutSettings.LabelPosition] property, you can layout the label associated with editor for data form.
+
+By default, the label will be positioned at left side of the editor.
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm">
+        <dataForm:SfDataForm.DefaultLayoutSettings>
+            <dataForm:DataFormDefaultLayoutSettings
+                LabelPosition="Left"/>
+        </dataForm:SfDataForm.DefaultLayoutSettings>
+    </dataForm:SfDataForm>
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.DefaultLayoutSettings.LabelPosition = DataFormLabelPosition.Top;
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Changing particular editors label position
+
+The label position of particular editor can be changed by using the [DataFormDefaultLayoutSettings.LabelPosition] property in `DataFormItem`, and it will be handled in the `GenerateDataFormItem` event.
```

Comment:
The label position of a particular editor can be changed by using the [DataFormDefaultLayoutSettings.LabelPosition] property in the `DataFormItem,` and it will be handled in the `GenerateDataFormItem` event.

Reply:
Content changed